### DISCUSSION TITLE
Is Postal susceptible to the log4j2 vulnerability (CVE-2021-44228)?

### DISCUSSION THREAD

**Discussion by willquill** [postalserver/postal#1772] [Help with using Postal] [answered]
Just wondering if I need to do anything if I am running the latest version _or any previous version_ of Postal with regard to CVE-2021-44228, the log4j2 vulnerability making waves in cyberspace right now.

**Comment by willpower232** [comment, marked as answer]
Postal uses no Java components so nothing to worry about in this case
